### DISCUSSION TITLE
My School not found when I try to select it from the app and hence my request for student pack was rejected

### DISCUSSION THREAD

**Discussion by pourjour** [community/community#55660] [GitHub Education]
### Hi everyone,

Hi everyone,
My School not found when I try to select it from the app and hence my request for student pack was rejected. I have no clue how to contact the github education team and I'd like for my problem to be solved.

**Comment by samus-aran** [comment]
Hi @pourjour 👋 

You are able to enter your School details if your school is not already available to select. We do suggest that you provide additional pieces of evidence to help the team verify the insituation.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope that helps.
